src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -503,4 +503,7 @@
   <data name="ResourceStreamEndedUnexpectedly" xml:space="preserve">
     <value>Resource stream ended at {0} bytes, expected {1} bytes.</value>
   </data>
+  <data name="SharedArgumentMissing" xml:space="preserve">
+    <value>Argument required after '/shared:'. </value>
```

Comment:
Got it -- missed the `:`. Maybe a simple "Value for argument `/shared:` must not be empty"?